test/get-own-property-symbols.js

```diff
@@ -318,5 +318,31 @@ wru.test([
       Point.prototype[Symbol.toStringTag] = 'Point';
       wru.assert(Object.prototype.toString.call(new Point) === '[object Point]');
     }
+  }, {
+    name: 'keyFor throws on non symbol',
+    test: function () {
+      var didTrow = false;
+      try {
+        Symbol.keyFor('not a Syumbol');
+      } catch(yay) {
+        didTrow = true;
+      }
+      wru.assert(didTrow, 'non symbols cannot be passed to keyFor');
```

Comment:
then the test has been bad for long time ... it looks like there are too
many developers working on the same diff though ... I agree it's a bit
messy right now.

On Tue, Jul 12, 2016 at 4:55 PM, Jake Champion <notifications@github.com>
wrote:

> In test/get-own-property-symbols.js
> <https://github.com/es-shims/get-own-property-symbols/pull/11#discussion_r70467705>
> :
>
> > @@ -318,5 +318,31 @@ wru.test([
> >        Point.prototype[Symbol.toStringTag] = 'Point';
> >        wru.assert(Object.prototype.toString.call(new Point) === '[object Point]');
> >      }
> > +  }, {
> > +    name: 'keyFor throws on non symbol',
> > +    test: function () {
> > +      var didTrow = false;
> > +      try {
> > +        Symbol.keyFor('not a Syumbol');
> > +      } catch(yay) {
> > +        didTrow = true;
> > +      }
> > +      wru.assert(didTrow, 'non symbols cannot be passed to keyFor');
>
> I haven't changed this line, not sure why it is in the diff ¯_(ツ)_/¯
>
> https://github.com/es-shims/get-own-property-symbols/blob/master/test/get-own-property-symbols.js#L330
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/es-shims/get-own-property-symbols/pull/11/files/6fcdf210dd2e6d1ab9819db1c5b3c15646d9d546#r70467705>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAFO9cToFLHhsBJajypxG1Mj18LoXecOks5qU7jagaJpZM4JKhKf>
> .
>